packages/scoreboard/lib/src/widgets/scoreboard/standingboard.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/material.dart';
+import '../../globals/themes.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+
+class StandingBoard extends StatefulWidget {
+  const StandingBoard({
+    Key? key,
+  })  : super(key: key);
+
+
+  @override
+  State<StandingBoard> createState() => _StandingBoardState();
+}
+
+class _StandingBoardState extends State<StandingBoard> {
+  var _list = [0, 1, 2];
+  var _images = [
+    'packages/scoreboard/assets/images_standingboard/GoldMedal.png',
+    'packages/scoreboard/assets/images_standingboard/SilverMedal.png',
+    'packages/scoreboard/assets/images_standingboard/BronzeMedal.png',
```

Comment:
@Yash-jar Use Icons.workspace_premium for the medals. Delete the png files you've added.

Reply:
deleted them